cpp/tests/strings/factories_test.cu

```diff
@@ -0,0 +1,140 @@
+/*
+ * Copyright (c) 2019, NVIDIA CORPORATION.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include <cudf/column/column_factories.hpp>
+#include <cudf/strings/strings_column_view.hpp>
+#include <cudf/types.hpp>
+
+#include <gtest/gtest.h>
+#include <gmock/gmock.h>
+#include <tests/utilities/cudf_test_fixtures.h>
+
+#include <vector>
+#include <cstring>
+
+
+struct FactoriesTest : public GdfTest {};
```

Comment:
```suggestion
struct FactoriesTest : public cudf::test::BaseFixture {};
```

The `GdfTest` fixture is deprecated. This should use the new one: https://github.com/rapidsai/cudf/blob/branch-0.10/cpp/tests/utilities/base_fixture.hpp#L35